plugins/config-cleanup.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 spec:
   version: "v0.2.0"
   shortDescription: Automatically clean up your kubeconfig
+  longDescription: Every cloud provider has their own utilities for adding kubernetes cluster credentials to your kubeconfig but most don't offer the ability to clean it up once the cluster is deleted. For those of us who launch and delete multiple clusters per day, it is useful to have an automated way to clean up old kubeconfig entries. This plugin will attempt to connect to each cluster defined in a context, if the connection succeeds then the user, cluster, and context entries are maintained in the result. Otherwise, the entries are removed.
```

Comment:
This should be “description”

Also please do line wrapping.

Reply:
Updated field name, fixed line wrapping, and shortened the description